### PR TITLE
Remove config.Pipeline.Name, already in the Piplines map key

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: service/internal/builder/exporters_builder.go [hunk 1]
```diff
@@ -125,7 +125,7 @@ func (exps Exporters) ToMapByDataType() map[config.DataType]map[config.Component
 
 type dataTypeRequirement struct {
 	// Pipeline that requires the data type.
```
Comment (codeboten) [inline service/internal/builder/exporters_builder.go:127]: I'm not sure the comment makes sense with the change.

File: config/config.go [hunk 2]
```diff
@@ -224,7 +224,6 @@ const (
 
 // Pipeline defines a single pipeline.
 type Pipeline struct {
-	Name       string
```
Comment (codeboten) [inline config/config.go:227]: Please add a note to the changelog

**Review by codeboten — COMMENTED** [review]
Please resolve the conflict.

File: service/internal/builder/exporters_builder.go (hunk 1 above)
Comment (bogdandrutu) [reply service/internal/builder/exporters_builder.go]: Done, simplified this more.

File: config/config.go (hunk 2 above)
Comment (bogdandrutu) [reply config/config.go]: Done.

**Review by codeboten — APPROVED** [review]
Thanks for addressing my comments

**Pull request merged**